pkg/kinflate/resource/kv.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resource
```

Comment:
Maybe we can move kv.go to a sub package inside resource.

Reply:
I guess we don't want to expose `kvPair`.

Reply:
It would have to go in an internal package

Reply:
Don't want to export this because don't see this being used outside resource pkg. So may be can have an internal package for it, but not so sure about that yet.